src/components/DropMenu/DropMenu.tsx

```diff
@@ -257,7 +258,7 @@ export interface IDropMenuProps extends StandardProps {
 	/** Called when an option is clicked, or when an option has focus and the
 			Enter key is pressed. */
 	onSelect: (
-		optionIndex: number | null,
+		optionIndex: any,
```

Comment:
Interesting, what was the reason for dropping to an `any` here?

Reply:
I think this was from a previous merge, I can't remember if I made this change or if it originated elsewhere since its on the DropMenu. I'll take another look quickly